dbuf-gen/src/generate/namespace.rs

```diff
@@ -0,0 +1,181 @@
+use std::hash::Hash;
+
+use super::{
+    lookup::{Cursor, LookupResult, NodeCursor},
+    node::Node,
+};
+
+struct Edge<'parent, Key: Eq + Hash, Value> {
```

Comment:
It's generally better to provide bounds only in `impl` blocks, not in the structure declarations